packages/components/src/Actions/Action/Action.component.js

```
@@ -114,8 +116,9 @@ function Action(props) {
 			{buttonContent}
 		</Button>
 	);
-
-	if (hideLabel || tooltip || tooltipLabel) {
+	if (!available) {
```

Adopt the fail fast principle by putting this at the top of the function please